src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/Semantic/SemanticTokensEditRequest.ts

```diff
@@ -0,0 +1,19 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import * as vscode from 'vscode';
+import { LanguageKind } from '../RPC/LanguageKind';
+
+export class SemanticTokensEditRequest {
+    public readonly razorDocumentUri: string;
+
+    constructor(
+        public readonly kind: LanguageKind,
+        razorDocumentUri: vscode.Uri,
```

Comment:
Is the following possible?

```suggestion
        public readonly razorDocumentUri: vscode.Uri,
```

Reply:
We need to `toString` URI's because otherwise they get serialized as objects.